templates/rabbitmq.conf.j2

```diff
@@ -1,3 +1,5 @@
 {{ ansible_managed | comment('erlang') }}
 
-{{ rabbitmq_config | encode_erlang(convert_bools=true, convert_nums=true) }}
+[
```

Comment:
Why did you add the square brackets here?

Reply:
I think there is a typo in the examples. The custom config should start like an array as it is in the default config in `defaults/main.yaml`. Then adding the square brackets in the template file is not required.